docs/tutorial/support.md

```diff
@@ -0,0 +1,93 @@
+# Electron Support
+
+## Finding Support
+
+If you have a security concern,
+please see the [security document](../../SECURITY.md).
+
+If you're looking for programming help,
+for answers to questions,
+or to join in discussion with other developers who use Electron,
+you can interact with the community in these locations:
+- [`electron`](https://discuss.atom.io/c/electron) category on the Atom
+forums
+- `#atom-shell` channel on Freenode
+- [`Atom`](https://atom-slack.herokuapp.com) channel on Slack
```

Comment:
Should this not be the `#electron` channel on the Atom Slack?

Reply:
Fixed b602714e06ae8a4121d3e1568ea4abbb4c5dfaa0

This error was already in README.md; all I did was propagate it in a new file :100: